clarity/src/vm/analysis/type_checker/contexts.rs

```diff
@@ -85,10 +92,16 @@ impl ContractContext {
             fungible_tokens: HashSet::new(),
             non_fungible_tokens: HashMap::new(),
             traits: HashMap::new(),
+            defined_traits: HashSet::new(),
+            all_traits: HashMap::new(),
             implemented_traits: HashSet::new(),
         }
     }
 
+    pub fn is_contract(&self, other: &QualifiedContractIdentifier) -> bool {
+        &self.contract_identifier == other
+    }
```

Comment:
Can you add a rustdoc to this method like

```
/// Checks if the contract represented by this `ContractContext` is the same contract referenced by `other`
```

Reply:
Added in 2b7ab449f